client/src/components/Dashboard.tsx

```diff
@@ -150,6 +150,7 @@ const Dashboard = () => {
                   rooms={rooms}
                   meetings={meetingsIndexed}
                   isDisabled={isDisabled}
+                  isJoinable={potentiallyDisabledGroup ? potentiallyDisabledGroup.isJoinable : true}
```

Comment:
Can be shorted to
potentiallyDisabledGroup?.isJoinable ?? true

Reply:
But mine is more readable, isn't it?